packages/cli/generators/datasource/index.js

```diff
@@ -63,6 +63,10 @@ module.exports = class DataSourceGenerator extends ArtifactGenerator {
   }
 
   setOptions() {
+    // remove not needed .env property
+    if (this.options.config) {
+      delete this.options?.env;
```

Comment:
Is this necessary for this PR to work?

Otherwise, I think this should be kept as a separate discussion/PR.

Reply:
This is actually coming from the master branch of my fork. I made another PR with these changes.